Tests/Testo/Invoice/InvRecurring/CronTokenRepositoryTest.php

```
@@ -0,0 +1,66 @@
+ * `invrecurring/cron` route's Bearer-token authentication. See
+ * docs/INVRECURRING_CRON_BEARER_AUTH_AUGUST_2026.md for the full
+ * before/after story this replaces (a hand-rolled `!==` comparison against
+ * a URL query parameter).
+ */
+#[Test]
+final class CronTokenRepositoryTest
+{
+    private function repositoryWithCronKey(string $cronKey): CronTokenRepository
+    {
+        /** @var SR&m\MockInterface $sR */
+        $sR = m::mock(SR::class);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Mockery' in method 'repositoryWithCronKey'. (undefined)

(StaticAccess)